apps/src/lib/node/ledger/protocol/mod.rs

```diff
@@ -174,7 +176,11 @@ where
                 ..Default::default()
             })
         }
-        _ => {
+        tx_type @ _ => {
+            tracing::error!(
+                "Attempt made to apply an unsupported transaction! - {:#?}",
+                tx_type
+            );
```

Comment:
`ProtocolTxType::EthereumEvents` with an empty vector of events was hitting this branch before, I've moved case that to be handled in the above branch. 

I'm not sure now if we should expect to hit this code path anymore. The only remaining tx types that would hit here are: `TxType::Wrapper`, non-Ethereum `TxType::Protocol` and `TxType::Decrypted(DecryptedTx::Undecryptable(_))`. We could replace this branch with returning an error like we do for `TxType::Raw(_)` i.e. `_ => Err(Error::TxTypeError)`.

Reply:
#313

Reply:
#313